primitives/src/runtime_api.rs

```diff
@@ -110,7 +110,7 @@
 //! All staging API functions should use primitives from `vstaging`. They should be clearly separated
 //! from the stable primitives.
 
-use crate::v2;
+use crate::*;
```

Comment:
Out of curiosity - isn't `use something::*` considered a bad practice?

Reply:
I'm not sure either if it's a bad practice or not, but to use only what you need is a valid point anyway -- fixed it.